README.md

```diff
@@ -1,20 +1,88 @@
 # Flow Data Provisioning Service
 
+The Flow Data Provisioning Service aims at providing a scalable and efficient way to access the execution state of past
+and live sporks, as well as implementing the Rosetta API standard.
```

Comment:
`Rosetta API specification, which could serve as the basis for a potential Coinbase integration of Flow.`

Reply:
```suggestion
and live sporks, as well as implementing the Rosetta API specification, which could serve as the basis for a potential Coinbase integration of Flow.
```